src/client/java/teammates/client/scripts/StatisticsPerInstitute.java

```diff
@@ -139,14 +137,10 @@ private StatsBundle generateStatsPerInstitute(
 
             String institute = getInstituteForStudent(student, allInstructors, allAccounts);
 
-            if (!institutes.containsKey(institute)) {
-                institutes.put(institute, new HashMap<Integer, Set<String>>());
-
-                institutes.get(institute).put(INSTRUCTOR_INDEX, new HashSet<String>());
-                institutes.get(institute).put(STUDENT_INDEX, new HashSet<String>());
-            }
-
+            institutes.computeIfAbsent(institute, key -> new HashMap<>()).put(INSTRUCTOR_INDEX, new HashSet<>());
+            institutes.get(institute).put(STUDENT_INDEX, new HashSet<>());
             institutes.get(institute).get(STUDENT_INDEX).add(student.getEmail().toLowerCase());
```

Comment:
same thing here

Reply:
sorry I just realized I overlooked a huge logic flaw. The modification done to this part and the part before this will result in a huge loss of student emails as the `institute.get(institute).put(STUDENT_INDEX, new Hashset<>())` will erase the record of student emails. 
Currently I cannot think of a way to replace this part of code using `computeIfAbsent` or `putIfAbsent` in a simpler manner than the original code. Lets leave this part and the part before (the other part you've changed in the same method) alone.